src/EditorFeatures/Core.Cocoa/Preview/PreviewPaneService.cs

```diff
@@ -97,13 +97,6 @@ private NSImage GetSeverityIconForDiagnostic(DiagnosticData diagnostic)
             var helpLinkUri = BrowserHelper.GetHelpLink(data);
             var helpLinkToolTip = BrowserHelper.GetHelpLinkToolTip(data.Id, helpLinkUri);
 
-            Guid optionPageGuid = default;
-            if (data.Properties.TryGetValue("OptionName", out _))
-            {
-                data.Properties.TryGetValue("OptionLanguage", out _);
-                throw new NotImplementedException();
-            }
-
             return new PreviewPane(
                 severityIcon: null,//TODO: Mac GetSeverityIconForDiagnostic(diagnostic),
                 id: data.Id, title: title,
```

Comment:
Need to pass in null for `optionPageGuid` below, or remove it and let the optional parameter take care of it.

Alternatively, could just remove that parameter from the `PreviewPane` constructor altogether since it's not used.

Reply:
Thanks! I completely missed it.